bears/generate_package.py

```diff
@@ -7,21 +7,13 @@
 import sys
 import time
 
+from pathlib import Path
```

Comment:
this is a Python standard library import, and belongs in the top group of imports